portality/migrate/3374_three_letter_language_code/operations.py

```diff
@@ -0,0 +1,43 @@
+from typing import TYPE_CHECKING
+
+from portality.lib import val_convert
+
+if TYPE_CHECKING:
+    from portality.models import Article
+
+lang_code_fn = val_convert.create_fn_to_isolang(is_upper=False)
+
+lang_map = {
+    'engilish': 'English',
+    'greek': 'gre',
+    'malay': 'may',
+}
+# some can't found by lookup
+country_name_map = {
+    'iran': 'IRN',
+    'south korea': 'KOR',
+}
+
+
+def update_article_code(article: 'Article'):
```

Comment:
I don't think this is the right migration.  

Article's get their language data set from the Journal record anyway, so we just need to run the `ArticleCleanupSync` background task (which can be done from the script `article_cleanup_sync.py` when we release).

Instead we need to apply this change to `Journal` and `Application` records, and then the Articles will update themselves when they are synchronised.